src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  margin-top: 0;
+}
+
+.form {
+  position: relative;
+  width: 100%;
+  margin-top: 20px;
+  display: inline-block;
+}
+
+.form-label {
+  display: inline-block;
+  width: 100%;
+}
+
+.top-search-bar {
+  height: 70px;
+  line-height: 70px;
+}
+
+.bottom-search-bar {
+  height: 42px;
+  line-height: 42px;
+}
+
+.search-image {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.big-search-image {
+  width: 19px;
+  height: 19px;
+  left: 26px;
+}
+
+.small-search-image {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+}
+
+.input-field {
+  outline: none;
+  font-family: avenir, sans-serif;
+  height: 100%;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background-color: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  padding: 0;
+  font-weight: 300;
+}
+
+.big-input-field {
+  font-size: 16px;
+  padding-left: 62px;
+}
+
+.small-input-field {
+  font-size: 14px;
+  padding-left: 33px;
+}
+
+.input-field:focus {
+  color: rgba(61, 78, 97, 1);
+  border-radius: 3px;
+  background: linear-gradient(180deg,
+  rgba(255, 255, 255, 1) 0%,
+  rgba(246, 246, 247, 1));
```

Comment:
this could be shorter if used with hex color codes, just as in the mock:
`linear-gradient(180deg, #FFFFFF 0%, #F6F6F7 100%);;`